packages/radix-ui-themes/esbuild-cjs.mjs

```diff
@@ -0,0 +1,22 @@
+import esbuild from 'esbuild';
+
+const dir = 'dist/cjs';
+
+const options = {
+  entryPoints: ['src/**/*.ts*'],
+  outdir: dir,
+  format: 'cjs',
+  target: 'es2020',
+  sourcemap: true,
```

Comment:
@benoitgrelard do we want to keep including source maps in the npm package?

See these issues:
https://github.com/radix-ui/themes/issues?q=is%3Aissue+is%3Aopen+source+map

What we have right now doesn't make sense to me:
- Source maps point to the original TypeScript files
- We ship source maps in the package
- Webpack complains that source maps point to files that don't exist

I think we should either drop them from non-dev builds or include the entire `src` contents into the npm package

Reply:
I think sourcemaps are good to have. We should keep them and publish the `src` on npm.

Reply:
Added